src/components/material/ReviewExternal.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { ExternalReview } from "../../core/dbc-gateway/generated/graphql";
+import ReviewHearts from "./ReviewHearts";
+import ReviewMetadata from "./ReviewMetadata";
+
+export interface ReviewExternalProps {
+  review: ExternalReview;
+}
+
+const ReviewExternal: React.FC<ReviewExternalProps> = ({ review }) => {
+  return (
+    <li className="review text-small-caption">
```

Comment:
Consider changing this to `<div>`. 

This is the classic list/item question. Should the item (`ReviewExternal`) be aware that it is always used within a list and thus use `<li>` as a wrapper or not and then use a `<div>` which then will result in more markup as the structure will become.

```html
<ul>
  <li>
     <div class="review"/>
  </li>
</ul>
 ```

Reply:
That is a very fair point. 
For now though, we don't need to reuse these reviews any other places in different contexts. So I will take this with me forward as a point to consider, but in this case the reviews are list items :)